src/pages/settings/Subscription/CardSection/BillingBanner/SubscriptionBillingBanner.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import * as Illustrations from '@components/Icon/Illustrations';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+import type IconAsset from '@src/types/utils/IconAsset';
+import BillingBanner from './BillingBanner';
+import type {BillingBannerProps} from './BillingBanner';
+
+type SubscriptionBillingBannerProps = Omit<BillingBannerProps, 'titleStyle' | 'subtitleStyle' | 'style' | 'brickRoadIndicator' | 'icon'> & {
+    isTrialActive?: boolean;
+    isError?: boolean;
+    icon?: IconAsset;
+};
+
+function SubscriptionBillingBanner({
+    title,
+    subtitle,
+    rightIcon,
+    icon,
+    isTrialActive,
```

Comment:
These are optional, do we want to set any default values ?

Reply:
Great idea, done